src/discover/moment_list.rs

```diff
@@ -0,0 +1,261 @@
+use makepad_widgets::*;
+
+live_design! {
+    import makepad_widgets::frame::*;
+    import makepad_widgets::label::*;
+    import makepad_widgets::list_view::ListView;
+    import makepad_widgets::image::*;
+
+    import crate::shared::styles::*;
+    import crate::shared::helpers::*;
+
+    IMG_DEFAULT_AVATAR = dep("crate://self/resources/img/default_avatar.png")
+
+    IMG_BANNER = dep("crate://self/resources/img/hero.jpg")
+    IMG_POST1 = dep("crate://self/resources/img/post1.jpg")
+    IMG_POST2 = dep("crate://self/resources/img/post2.jpg")
+
+    Hero = <Frame> {
+        walk: {width: Fill, height: Fit}
+        banner = <ImageFrame> {
```

Comment:
Why do we use ImageFrame here instead of Image? Is it to be able to be used as background?

Reply:
thats right, I could not get it to work with the new Image widget